backend/app/processing/kb.py

```
@@ -0,0 +1,296 @@
+import os
+import json
+import numpy as np
+import faiss
+import datetime
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)